src/main/java/net/redstoneore/legacyfactions/cmd/CmdFactionsDescription.java

```diff
@@ -31,13 +32,16 @@ public void perform() {
             return;
         }
 
-        // since "&" color tags seem to work even through plain old FPlayer.sendMessage() for some reason, we need to break those up
-        // And replace all the % because it messes with string formatting and this is easy way around that.
-        myFaction.setDescription(TextUtil.implode(args, " ").replaceAll("%", "").replaceAll("(&([a-f0-9klmnor]))", "& $2"));
+        // Replace all the % because it messes with string formatting and this is easy way around that.
+        if(Conf.allowColorCodesInFaction) {
```

Comment:
I'm not too picky, but in some cases people will want all the code to use the same style (in the case; space after the if, and spaces before and after 'else').

You don't have to as the style guide of this project is a mess and will need to be formatted later - but just something to keep in mind 😄